src/dft/backends/rocfft/CMakeLists.txt

```diff
@@ -41,10 +41,22 @@ target_include_directories(${LIB_OBJ}
 target_compile_options(${LIB_OBJ} PRIVATE ${ONEMKL_BUILD_COPT})
 
 find_package(HIP REQUIRED)
-find_package(rocfft REQUIRED)
+find_package(rocfft 1.0.21 REQUIRED)
```

Comment:
Might be nice to have a note that this corresponds to the ROCm 5.4.3.

Reply:
Done in https://github.com/oneapi-src/oneMKL/pull/471/commits/6722aa501f864ec70ab036faebda1bad3f3ee1b7